tests/Tester.py

```diff
@@ -24,24 +28,32 @@ def run_tests():
     if not os.path.exists("../results/"):
         os.makedirs("../results/")
     results_file_path = "../results/{}.tsv".format(datetime.datetime.now().strftime("%Y-%m-%d_%H-%M-%S"))
+    log.info("Results will be stored in {}".format(results_file_path))
 
     with open(results_file_path, 'w', newline='') as f:
-        writer = csv.writer(f, delimiter='\t',)
-        writer.writerow(["start", "goal", "heuristic", "time_seconds", "nodes_expanded", "path_length", "path"])
+        writer = csv.writer(f, delimiter='\t', )
```

Comment:
Not sure how we missed this but there probably shouldn't be a trailing comma here.